src/datasources/entity-ds/queries/queryNodes.ts

```diff
@@ -64,7 +80,9 @@ export const queryNodes = async (client: ClientDelegate, filter: API.Filter): Pr
             node.netBiosDomain,
             node.operatingSystem,
             node.lastCapsdPoll,
-            node.categories ? node.categories.map(cat => cat.name) : undefined,
+            ifIndex || '',
+            ipAddress,
+            node.categories ? node.categories.map(cat => cat.name) : ''
```

Comment:
Is it going to mess anything up that the ordering of these changed?

Reply:
The returned rows should match the order of the `columns` above, so this was actually a bug since these items were in the columns expected but not in the rows, so `Categories` data was where `Primary SNMP ifIndex` was expected.